iOS/Source/Root/Extension/Extensions.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+import UIKit
+
+extension UIColor {
```

Comment:
I always create one file for each extension. That might make it easier to find it back later.